pkg/executables/kubectl.go

```diff
@@ -1364,12 +1366,24 @@ func (k *Kubectl) GetResource(ctx context.Context, resourceType string, name str
 	return found, err
 }
 
-func (k *Kubectl) getObject(ctx context.Context, resourceType, name, namespace, kubeconfig string, obj client.Object) error {
-	stdOut, err := k.Execute(ctx, "get", "--namespace", namespace, resourceType, name, "-o", "json", "--kubeconfig", kubeconfig)
+// GetObject performas a GET call to the kube API server authenticating with a kubeconfig file
```

Comment:
you made me look up `performas` and see if it's a proper word. showing in multiple places